com.unity.shadergraph/Editor/Importers/ShaderGraphAssetPostProcessor.cs

```diff
@@ -88,7 +88,7 @@ static void OnPostprocessAllAssets(string[] importedAssets, string[] deletedAsse
             }
 
             // moved or imported subgraphs or HLSL files should notify open shadergraphs that they need to handle them
-            var changedFiles = movedAssets.Union(importedAssets)
+            var changedFiles = movedAssets.Concat(importedAssets).Concat(deletedAssets)
```

Comment:
Union is the same thing as Concat.Distinct, and we already have a Distinct below.